pyproject.toml

```diff
@@ -7,7 +7,7 @@ license = "MIT"
 
 [tool.poetry.dependencies]
 python = "^3.7"
```

Comment:
Good opportunity to bump to a much more recent version of Python e.g. 3.10?  3.7 not got that much life left in it.

Reply:
_Although_ you want to make sure you have the specified version of Python available in your deployment enviroment.

Reply:
Noted.